mfa.go

```diff
@@ -11,10 +11,11 @@ import (
 )
 
 var mfaLoginField = plugins.LoginField{
-	Label:       "MFA Token",
-	Name:        plugins.MFALoginFieldName,
-	Placeholder: "(optional)",
-	Type:        "text",
+	Label:         "MFA Token",
+	Name:          plugins.MFALoginFieldName,
+	Placeholder:   "(optional)",
+	Type:          "text",
+	autocomplete:  "one-time-code",
```

Comment:
That would be a private field and does not exist.

Reply:
Oh, thank you for your answer!
Is there a way, to add `autocomplete= "one-time-code"` into the input tag?

Like here:
```html
<input  autocomplete="one-time-code" onclick="" id="simple-mfa-token" name="simple-mfa-token" placeholder="(optional)" type="text" class="form-control">
```

Only adding the autocomplete here, wouldn't solve the problem.?

https://github.com/Luzifer/nginx-sso/blob/cae187e39ccc40c97971ef7f8a4cadb40d73f5e5/plugins/auth.go#L50-L56
---
Sorry for my question, it is my first time working with Go.